sqe-api-server/HttpControllers/RoiController.cs

```diff
@@ -108,7 +108,7 @@ public RoiController(IRoiService roiService, IUserService userService)
         [HttpPut("v1/editions/{editionId}/rois/batch")]
         public async Task<ActionResult<UpdatedInterpretationRoiDTOList>> UpdateRois(
             [FromRoute] uint editionId,
-            [FromBody] InterpretationRoiDTOList updateRois)
+            [FromBody] UpdateInterpretationRoiDTOList updateRois)
```

Comment:
I think if you were using the SignalR methods in sqe-signalr.ts for backend calls (instead of HTTP), you would get the integrated safety you are looking for, but I understand that HTTP requests are perhaps easier to debug.  

Very soon, I will integrate an openapi generator (https://github.com/OpenAPITools/openapi-generator), probably into the GitHub actions, so that we generate API libraries.  That could be an option as well, since among other libs I can create a typescript library and push it to NPM for each update.